### ISSUE TITLE
Defined terms no longer being captured correctly

### ISSUE THREAD

**Issue by goose-life** [laws-africa/indigo#1564]
Fix bug introduced by edf331d887e0cce68c1153207d917a3194a36db4

We do want to pick up terms in single quotes, but we don't want to miss half of 
"**Women**’s Police Network Sub-sub Committee", or "**member of the victim**’s household"
(https://edit.laws.africa/works/akn/aa-sadc/act/2018/southern-african-regional-police-chiefs-co-operation-organisation/, https://edit.laws.africa/works/akn/ls/act/2012/6/)

In other words, either we ditch single-quote definitions or we update `term_re` to look for matching pairs of quotes:
`"+"`, `“+”`, `'+'`, `‘+’`, `«+»`

**Comment by longhotsummer** [comment]
Matching quotes for the win, I think. Something like this perhaps:

`r'("[^"]+")|(«[^«]+«)|...'`

**Comment by goose-life** [comment]
hmm. this approach is giving me uphill because we capture the term itself and use match.group(1) when marking the definition.
but the right group would be match.group(2) if we had `"…"|'…'`and the term was in `''`s.

**Comment by longhotsummer** [comment]
ah :( you'll have to change the group that is used (which will be variable)

short term: remove apostrophe matches

**Comment by goose-life** [comment]
interested to see what you think of my attempt at a variable group! but happy to just ditch single quotes if that doesn't work for you, lmk
